packages/matchbox/src/components/Slider/tests/Slider.test.js

```diff
@@ -35,6 +35,11 @@ describe('Slider component', () => {
     expect(slider.find('.Handle')).toHaveAttributeValue('id', 'test-id');
   });
 
+  it('should render an aria-controls attribute ', () => {
```

Comment:
I think we can remove this unused import -> `import { action } from '@storybook/addon-actions';`